src/Tabs/TabsWrapper.php

```diff
@@ -206,7 +206,7 @@ function twAddFrameTab(tabsid, label, url) {
       temp = panelId;
       return panelId;
     } else {
-      setTimeout('alert("Fee sheet tab is already opened")', 5);
+      setTimeout(function(){alert("Fee sheet tab is already opened");}, 5);
```

Comment:
please lets use our timed BS alert in utility.js

`let msg = <?php echo xlj("Fee sheet tab is already opened") ?>;`
`(async (time) => {
     await asyncAlertMsg(msg, time, 'warning');
 })(3000).then(rtn => { "could use to wrapper rest of condition" });`

could also call:
`syncAlertMsg(message, timer = 5000, type = 'danger', size = '')`

Reply:
@sjpadgett  Is this the expected behaviour?

![Screenshot (141)](https://user-images.githubusercontent.com/55142272/79267931-8f820800-7eb7-11ea-9183-e2605e87e68e.png)

Reply:
Works much better don't you think? Also instead of danger wouldn't warning be more appropriate color?
 These alert functions will always return a promise if you wanted to do some after items after the alert is disposed by time out or x-out. Could be handy in validation use case and many others. Just something to put in your toolbox.

Looks okay to me and i'll let @stephenwaite finish the review but, nice job.

Reply:
@sjpadgett Thanks for the review, will make the final changes. Also learned a new thing today thanks !! 😊